components/edit-collective/sections/Host.js

```diff
@@ -132,12 +131,12 @@ class Host extends React.Component {
           {collective.stats.balance === 0 && (
             <Fragment>
               <p>
-                <Button bsStyle="primary" type="submit" onClick={() => this.changeHost()} className="removeHostBtn">
```

Comment:
It's good to keep the `type="submit"` if already set, makes things explicit in the DOM